python/triqs_dft_tools/converters/plovasp/plotools.py

```diff
@@ -85,7 +85,7 @@ def check_data_consistency(pars, el_struct):
 # Check that ions inside each shell are of the same sort
     for sh in pars.shells:
         max_ion_index = max([max(gr) for gr in sh['ions']['ion_list']])
-        assert max_ion_index < el_struct.natom, "Site index in the projected shell exceeds the number of ions in the structure"
+        assert max_ion_index <= el_struct.natom, "Site index in the projected shell exceeds the number of ions in the structure"
```

Comment:
Can you quickly comment on this? Was this wrong previously? I thought "<" is correct as `max_ion_index` starts from 0 whereas `el_struct.natom` starts at 1?

Reply:
I am not sure about this, maybe was a debug ... but if it works also without then should be fine.